foreman/data_refinery_foreman/surveyor/test_end_to_end.py

```diff
@@ -611,7 +608,7 @@ def test_sra_redownloading(self):
                             break
 
             # There's a chance that the processor job with a missing
-            # file deletes itself before the last downloader job
+            # file is aborted before the last downloader job
             # completes, therefore just check that there's at least 3
             # processor jobs.
```

Comment:
We no longer have to check for at least three, because there will always be 4 since the one with a missing file will not delete itself.

Reply:
I mean that we can just check for exactly 4 jobs here.